icu4j/main/tests/core/src/com/ibm/icu/dev/test/rbbi/RBBITestMonkey.java

```diff
@@ -133,28 +180,32 @@
             fAnySet           = new UnicodeSet("[\\u0000-\\U0010ffff]");
 
 
-            fSets       = new ArrayList();
-            fSets.add(fCRLFSet);
-            fSets.add(fControlSet);
-            fSets.add(fExtendSet);
-            fSets.add(fRegionalIndicatorSet);
+            fSets         = new ArrayList();
+            fClassNames   = new ArrayList();
```

Comment:
This line and the next could be deleted if the initialization is moved to the base class.
(fSets can't be done that way as easily, because it's not currently declared in the base class.)